Automattic/jetpack
# [Plugin] Backup: Update initial backup screen

Whenever the initial backup is about to start after the initial purchase and plugin connection, show the backup running screen with 0% progress instead of a welcome message.

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
1- On a new site, add Jetpack Backup 
2- Purchase a plan for it. (no need to use the Jetpack plugin)
2- Setup the connection from the Backup plugin
3- Confirm that the Backup in progress page is shown instead of the `Welcome to Jetpack Backup!` page.
